test/unit/rules/lint-prettier-test.js

```diff
@@ -8,6 +8,9 @@ generateRuleTests({
   groupingMethod: describe,
   testMethod: it,
   plugins: [plugin],
+  config: {
+    singleQuote: true,
+  },
```

Comment:
Can we use `config: true` as the default here since that's how most people will use this rule? Then just use this specific options config in one of the individual test cases.